ssn/integrated/proposed-server-specification.md

```diff
@@ -528,3 +528,119 @@ When operating a GET at http://www.w3.org/ns/ssn/systems/xyz with a Accept heade
 
 When operating a GET at http://www.w3.org/ns/ssn/systems/xyz with a Accept header that is compatible with `text/turtle`, 303 redirect to http://www.w3.org/ns/ssn/systems.ttl
 
+
+# Implementation example: with Apache Server .htaccess
+
+
+```
+DirectoryIndex index.html
+
+RewriteEngine On
+RewriteBase  /ns/sosa/
+
+
+# Documents
+
+<Files ~ "sosa\.rdf$">
+    Header set Content-Type "application/rdf+xml"
+    Header set Content-Disposition "filename= sosa.rdf"
```

Comment:
According to the [grammar in the RFC spec that defines Content-Disposition](https://tools.ietf.org/html/rfc6266#section-4.1), the value should include the disposition type:

```apache
Header set Content-Disposition "inline; filename=sosa.rdf"
```

I'm not exactly clear as to which disposition type we want to use: `inline` or `attachment`. I guess I would prefer `inline` which in theory allows the user to view the document in his browser (as opposed to saving it directly). But then, if the goal was actually to force the browser to save the document, `attachment` is the value we want here. I don't think all browsers support the disposition type in any case.

Reply:
Let's go with `inline`, to me it's better user experience to get the document directly. 

A quick 2013 search led me to that 2013 report http://www.jtricks.com/bits/content_disposition.html that lists support for that header in common browsers -> apparently all ok

Reply:
Good, let's go for `inline`.

FWIW, "support" was not the right term here. What I meant was that the browser may not render the document even though `inline` is used, simply because it does not support the MIME type. That typically happens with Edge, which offers to save the Turtle version no matter what. But that's indeed orthogonal to supporting the `Content-Disposition` directive!